packages/icons-gcp/scripts/icon-setup.js

```
@@ -0,0 +1,192 @@
+/**
+ * GCP Icon Setup Script (Node.js version)
+ * Run with: node packages/icons-gcp/scripts/icon-setup.js
+ */
+const fs = require('fs-extra');
```

ESLint

[error] A `require()` style import is forbidden.

(`@typescript-eslint/no-require-imports`)

---

[error] 'require' is not defined.

(no-undef)